jiant/models.py

```diff
@@ -881,7 +886,9 @@ def _single_sentence_forward(self, batch, task, predict):
         classifier = self._get_classifier(task)
         logits = classifier(word_embs_in_context, sent_mask)
         out["logits"] = logits
-        out["n_exs"] = get_batch_size(batch)
+        out["n_exs"] = (
```

Comment:
any reason not to have `get_batch_size` return a tensor (and maybe even a cuda tensor) rather than repeat this everywhere ?

Reply:
Good point